app/upgrades/testnet/v8_0_0/upgrades.go

```diff
@@ -0,0 +1,71 @@
+package v8_0_0 //nolint:revive,stylecheck
+
+import (
+	assetkeeper "github.com/comdex-official/comdex/x/asset/keeper"
+	assettypes "github.com/comdex-official/comdex/x/asset/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+)
+
+func UpdateDenomAndAddAsset(ctx sdk.Context, assetKeeper assetkeeper.Keeper) {
+	asset, found := assetKeeper.GetAsset(ctx, 17)
+	if found {
+		asset.Denom = "ibc/50EF138042B553362774A8A9DE967F610E52CAEB3BA864881C9A1436DED98075"
+		assetKeeper.SetAsset(ctx, asset)
+	}
+
+	assetGDAI := assettypes.Asset{Name: "GDAI", Denom: "ibc/109DD45CF4093BEB472784A0C5B5F4643140900020B74B102B842A4BE2AE45DA", Decimals: sdk.NewInt(1000000000000000000), IsOnChain: false, IsOraclePriceRequired: true, IsCdpMintable: false}
```

Comment:
is it GDAT or grav.dai ? what's finalised ?

Reply:
GDAI is finalized.